src/core/chp.jl

```diff
@@ -87,7 +88,11 @@ can_net_meter::Bool = false
 can_wholesale::Bool = false
 can_export_beyond_nem_limit::Bool = false
 can_curtail::Bool = false
-# emissions_factor_lb_CO2_per_mmbtu::Float64
+fuel_renewable_energy_pct::Union{Nothing,Float64} = nothing
+emissions_factor_lb_CO2_per_mmbtu::Union{Nothing,Float64} = nothing
```

Comment:
The CHP doc string is incorrect for emissions inputs

Reply:
Resolving. See updates in https://github.com/NREL/REopt.jl/commit/7c7fba7d1e0da2e4b752e23f0c0bd5b3a58ba9fe for resolution.